### ISSUE TITLE
connection close?

### ISSUE THREAD

**Issue by gauravphoenix** [eriklumme/liquibase-from-java#1]
Thanks for providing a great example- just curious, I am not able to see any JDBC connection close statement, since you are creating a new connection manually by calling getConnection, shouldn't the connection be manually closed too?

**Comment by eriklumme** [comment]
You're right. When running this is a standalone tool, the connections seem to be cleaned up quite quickly, so it's not much of an issue. That being said, manually closing them is the right thing to do, so I've updated the code, thanks!

**Comment by gauravphoenix** [comment]
Have you tried using connection manager like HikariCp etc with clickhouse?
